docs/conceptual/list.sumby['t,^u]-function-[fsharp].md

```diff
@@ -30,29 +30,42 @@ List.sumBy projection list
 ```
 
 #### Parameters
-*projection*
-Type: **'T -&gt; ^U**
-
+*projection* Type: **'T -&gt; ^U**
 
 A function that transforms the list elements into the type to be summed.
 
 
-*list*
-Type: **'T**[list](https://msdn.microsoft.com/library/c627b668-477b-4409-91ed-06d7f1b3e4a7)
-
+*list* Type: **'T**[list](https://msdn.microsoft.com/library/c627b668-477b-4409-91ed-06d7f1b3e4a7)
 
 The input list.
 
 
+## Return Value
+The sum of the results of applying the projection function to each element of the list.
+
+
+## Exceptions
+May throw an [OverflowException](https://msdn.microsoft.com/library/system.overflowexception.aspx) due to arithmetic overflows.
+
 
-**The resulting sum.**
 ## Remarks
+The results are summed using the **[checked + operator](https://msdn.microsoft.com/visualfsharpdocs/conceptual/checked.[-p-][%5Et1%2c%5Et2%2c%5Et3]-function-[fsharp])** and **Zero** property associated with the summed type.
+
 This function is named **SumBy** in compiled assemblies. If you are accessing the function from a .NET language other than F#, or through reflection, use this name.
 
-**The following code example illustrates the use of [List.sum](https://msdn.microsoft.com/library/54d47fe3-5ecf-4883-beb5-e915342a17f9), List.sumBy, and [List.average](https://msdn.microsoft.com/library/2b9a627b-106d-4548-8c4c-ab5058b8f8e1).**
```

Comment:
Again, that's where it already is, unless I'm doing something wrong in using Github.